src/extra_functions.jl

```diff
@@ -107,17 +107,7 @@ derivative(::typeof(!=), ::NTuple{2, Any}, ::Val{i}) where {i} = 0
 derivative(::typeof(expinti), args::NTuple{1,Any}, ::Val{1}) = exp(args[1])/args[1]
 
 @register_symbolic SpecialFunctions.expint(nu, z)
-# https://en.wikipedia.org/wiki/Exponential_integral#Derivatives
```

Comment:
these removed were a rebase issue?

Reply:
No, they're already defined in a different file using DiffRules. This was causing method overwriting and failing precompilation.

Reply:
wouldn't that also register symbolic?

Reply:
No, it only defines the derivative rule. Most `SpecialFunctions` functions are "registered" in SU, but Mattia added these here. I'll move them back in the breaking release.